src/PocketMine.php

```diff
@@ -166,7 +166,7 @@ function check_platform_dependencies(){
 	 * @return void
 	 */
 	function emit_performance_warnings(\Logger $logger){
-		if(PHP_DEBUG !== 0){
+		if(ZEND_DEBUG_BUILD !== false){
```

Comment:
could just write `if(ZEND_DEBUG_BUILD)` though, since it's always a bool?

Reply:
You are right, fixed now. Thank you.